### PR TITLE
Add navigation with numbers

### LINKED ISSUE #522: Move the panel with number like lazygit
**Is your feature request related to a problem? Please describe.**
I get used to the feature switching panel in lazygit using number, but it's very frustrating to focus the different panel using mouse.

**Describe the solution you'd like**
In lazygit, you can change the focus pressing the number but lazydocker does not have that feature. Would it be possible to add that feature? So, if you press `1`, `Project` panel focus, `2`, `Services` panel focus, etc.

**Describe alternatives you've considered**
--

**Additional context**

### PR DESCRIPTION
**Pull request by peauc**
Fix #522 
Like lazygit we can navigate with numbers.

### CODE REVIEW

File: pkg/i18n/english.go [hunk 1]
```diff
@@ -128,6 +128,13 @@ type TranslationSet struct {
 	LcNextScreenMode string
 	LcPrevScreenMode string
 	FilterPrompt     string
+
+	FocusProjects   string
+	FocusServices   string
+	FocusContainers string
+	FocusImages     string
+	FocusVolumes    string
+	FocusNetwork    string
```
Comment (jesseduffield) [inline pkg/i18n/english.go:137]:
```suggestion
	FocusNetworks    string
```
Should be plural for consistency

**Review by jesseduffield — CHANGES_REQUESTED** [review]
Nice one, just one comment

File: pkg/i18n/english.go (hunk 1 above)
Comment (peauc) [reply pkg/i18n/english.go]: @jesseduffield changes has been addressed.

**Comment by jesseduffield** [comment]
Looks good @peauc. Can you squash your commits down to one?

**Comment by peauc** [comment]
@jesseduffield of course, I do not mind.

**Comment by peauc** [comment]
alright its done :) @jesseduffield

**Comment by peauc** [comment]
Small nudge @jesseduffield

**Review by jesseduffield — APPROVED** [review]
LGTM

**Comment by reconcrete** [comment]
For some reason, this feature is not included if you install `lazydocker` with brew tap version

If anyone going to face it, try installing manually with `go` cli

https://github.com/jesseduffield/lazydocker#go
https://github.com/jesseduffield/lazydocker#manual

**Comment by dalbir-singh** [comment]
> For some reason, this feature is not included if you install `lazydocker` with brew tap version
> 
> If anyone going to face it, try installing manually with `go` cli
> 
> https://github.com/jesseduffield/lazydocker#go https://github.com/jesseduffield/lazydocker#manual

Can confirm it doesn't work with the brew tap version, but if you were to install go and then manually install lazydocker it works with number shortcuts shown. Frustrating because I like to stick to brew to manage everything... I reverted back to the brew version (without working number shortcuts) and hope it will work one day in the future.

**Comment by vincenthongzy** [comment]
installing the binary using curl also does not include this feature

```
curl https://raw.githubusercontent.com/jesseduffield/lazydocker/master/scripts/install_update_linux.sh | bash

lazydocker --version
Version: 0.24.1
Date: 2024-11-23T06:32:15Z
BuildSource: binaryRelease
Commit: be051153525b018a46f71a2b2ed42cde39a1110c
OS: linux
Arch: amd64
```

**Pull request merged**